uktrade/data-hub-frontend
# Add helper text to the Investments- Opportunities tab

## Description of change

This PR adds some additional text to the UK opportunities tab, to help stop unapproved opportunities from being logged. 

## Test instructions

Go to Investments and click on the UK opportunities tab. You should see some additional text appear between the header and the tabs. This text should disappear when you click on either of the other two tabs and the email address should open up a blank email addressed to the email address when clicked.

## Screenshots
### Before

<img width="1134" alt="Screenshot 2021-12-22 at 13 05 50" src="https://user-images.githubusercontent.com/46787754/147097382-59872b56-0390-4d43-bf39-1d31083e609a.png">


### After

<img width="1124" alt="Screenshot 2021-12-22 at 13 06 21" src="https://user-images.githubusercontent.com/46787754/147097428-2ead4f31-5c53-4bed-a4e8-51843c2e6bd2.png">


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
